### DISCUSSION TITLE
Unable to Interact with Server via HTTP in Production, Works Fine in Development

### DISCUSSION THREAD

**Discussion by XPein** [tauri-apps/tauri#11082] [Q&A]
build my app with "cargo tauri dev", application successfully interacts with the server via HTTP. However, after deploying the same code to the production environment, HTTP communication with the server fail:failed to fetch . The codebase is identical in both environments, so the issue may be related to product configurations.

**Comment by FabianLars** [comment]
If this is tauri v1, try https://tauri.app/v1/api/config/#securityconfig.dangeroususehttpscheme

If it's v2, or the above didn't help, try a debug build via `cargo tauri build --debug` (output in target/debug/) and check the devtools console and network tab for errors.
